RxSwift/Schedulers/CurrentThreadScheduler.swift

```diff
@@ -51,7 +51,11 @@ public class CurrentThreadScheduler : ImmediateSchedulerType {
     private static var isScheduleRequiredKey: pthread_key_t = { () -> pthread_key_t in
         let key = UnsafeMutablePointer<pthread_key_t>.allocate(capacity: 1)
         defer {
+#if swift(>=4.1)
             key.deallocate(capacity: 1)
```

Comment:
These Swift branches currently do the same, I think you meant `key.deallocate()` in this case?